SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -530,7 +530,14 @@ void USpatialReceiver::HandleActorAuthority(const Worker_AuthorityChangeOp& Op)
 		{
 			if (USpatialActorChannel* ActorChannel = NetDriver->GetActorChannelByEntityId(Op.entity_id))
 			{
-				ActorChannel->ClientProcessOwnershipChange(Op.authority == WORKER_AUTHORITY_AUTHORITATIVE);
+				if (Op.authority == WORKER_AUTHORITY_AUTHORITATIVE)
+				{
+					ActorChannel->ClientProcessOwnershipChange(true);
+				}
+				else if (Op.authority == WORKER_AUTHORITY_NOT_AUTHORITATIVE)
+				{
+					ActorChannel->ClientProcessOwnershipChange(Op.authority == WORKER_AUTHORITY_AUTHORITATIVE);
```

Comment:
Isn't this always false since it's an elseif and the first if is checking for the same thing?

Reply:
Good point, I'll change that to explicit false.